src/pages/home/sidebar/SidebarScreen/BaseSidebarScreen.js

```diff
@@ -62,10 +63,51 @@ class BaseSidebarScreen extends Component {
         Welcome.show({routes, showCreateMenu: this.showCreateMenu});
     }
 
+    componentDidUpdate(prevProps) {
+        if (!this.didBecomeInactive(prevProps)) {
+            return;
+        }
+
+        // Hide menu manually when other pages are opened using shortcut key
+        this.hideCreateMenu();
+    }
+
+    didBecomeInactive(prevProps) {
+        // When Report page is just opened
+        if (!this.props.isDrawerOpen && prevProps.isDrawerOpen) {
+            return true;
+        }
+
+        // When other page is just opened
+        if (!this.props.isFocused && prevProps.isFocused) {
+            return true;
+        }
+
+        return false;
+    }
+
+    isInactive() {
```

Comment:
is Inactive what? Can we think of a better name? Functions are named on what they do not where they are called.

Reply:
cc: @Beamanator

Reply:
Agreed, what about `isSidebarScreenHidden`?